doc/sphinx/addendum/type-classes.rst

```diff
@@ -289,8 +298,24 @@ declares each projection as an instance.
 .. warn:: Ignored instance declaration for “@ident”: “@term” is not a class
 
    Using this ``:>`` syntax with a right-hand-side that is not itself a Class
-   has no effect (apart from emitting this warning). In particular, is does not
-   declare a coercion.
+   has no effect (apart from emitting this warning).
+
+.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.
+
+   :g:`Class >` will behave as :g:`Record >` in future versions,
+   declaring the construcotr a coercion.
+
+.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.
+
+   :g:`Record >` make the constructor a coercion, but there is no
+   constructor for singleton classes.
```

Comment:
```suggestion
.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.

   :g:`Class >` will behave as :g:`Record >` in future versions,
   declaring the constructor a coercion.

.. warn:: 'Class >' currently does nothing, a coercion will be introduced in future versions.

   :g:`Record >` make the constructor a coercion, but there is no
   constructor for singleton classes.
```
Messages are identical, combine.  Bigger question: what should the user do to make these warnings go away?  If there's no way for the user to fix this, what's the benefit of these warnings?  I would avoid the "in the future" wording.

Reply:
Thanks, that's a mistake (bad copy/paste), fixed. I added a sentence on how to avoid the warning.